examples/tests/valid/dynamo.w

```diff
@@ -0,0 +1,94 @@
+bring "@cdktf/provider-aws" as tfaws;
```

Comment:
This test only works on AWS, so this test might need a `skip-sim` annotation (how do I do this?)

Reply:
```
/*\
skip: true
\*/
```

This works in Hangar, but not via the cli, I also thought about it today when I wanted to skip a test, I think we should think of a way to unify all of those skipping scenarios 🤔